pagopa/pn-frontend
# Fix/pn 6185

## Short description
insert border as design
## List of changes proposed in this pull request
- insert border and border radius
- delete index in groups map and delete divider

## How to test
when you open dialog of group , in apy key page, you will see  the groups inside borders
